src/structures/MessageCollector.js

```diff
@@ -49,6 +56,7 @@ class MessageCollector extends Collector {
       this.emit('message', message);
     };
     this.on('collect', this._reEmitter);
+    this.on('remove', this._reEmitter);
```

Comment:
Why are you emitting an event documented as `Emitted when the collector receives a message.` when a message gets removed from the collector?